docs/source/_static/depthai_env_setup.sh

```diff
@@ -177,3 +199,127 @@ else
     echo "ERROR: Host not supported"
     exit 99
 fi
+
+echo "Finished installing global libraries."
+
+# add depthai working dir to .bashrc if its not already there
+COMMENT='# Home directory of depthai and depthai app, enables to run <run_demo> in terminal'
+BASHRC="$HOME/.bashrc"
+ADD_DEPTHAI_TO_PATH='export PATH=$PATH'":$WORKING_DIR"
+ADD_LUXONIS_DEPTHAI_HOME='export LUXONIS_DEPTHAI_HOME='"$WORKING_DIR"
+
+# add to .bashrc only if it is not in there already
+write_in_file "$COMMENT" "$BASHRC"
+write_in_file "$ADD_DEPTHAI_TO_PATH" "$BASHRC"
+write_in_file "$ADD_LUXONIS_DEPTHAI_HOME" "$BASHRC"
```

Comment:
This brings in ALL the files under PATH.

I'd rather have the following structure:
 - [working dir]
     - depthai (cloned)
     - entrypoint/
     - venv/

Where depthai remains git clone folder as is.
venv is created by script
and `entrypoint/` is also created by script and a SIMPLE entry point to a KNOWN location is created - eg to Launcher

Other option is slightly reversing, but we need to create then a "entrypoint" folder in `depthai` repo, with these scripts and those then need to be given paths to venv and interpreter, etc... (which is also fine)

In the latter case, its slightly more extensible, as then we don't have to rely on launcher as the entry point.

Later case:
 - [working dir]
     - depthai (cloned)
         - entrypoint
             - depthai_launcher
             - ...
     - venv

In this case, entrypoint scripts rely on being passed some upfront information like `LUXONIS_DEPTHAI_HOME` and/or `LUXONIS_DEPTHAI_HOME_VENV`and then use that to launch the app